app/views/validation_requests/index.html.erb

```diff
@@ -55,6 +55,10 @@
         <% if @validation_requests["data"]["pre_commencement_condition_validation_requests"].present? %>
           <%= render "pre_commencement_condition_validation_requests" %>
         <% end %>
+
+        <% if @validation_requests["data"]["heads_of_terms_validation_requests"].present? %>
```

Comment:
I wonder if for this whole section we could just do something like:

```erb
<% @validation_requests["data"].sort.each do |validation_request| %>
  <%= render validation_request %>
<% end %>
```

It depends how much we care about the order of the list entries, I suppose.